/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform
c0909a-s17.ufhpc
Tue May  3 13:39:15 EDT 2022
Primary node: c0909a-s17
Primary TCP port: 20602
Secondary nodes: 
Running "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py" on each node...
[33mWARNING:[0m underlay of /usr/bin/nvidia-smi required more than 50 (484) bind mounts
/.singularity.d/env/10-docker2singularity.sh: line 2: uname: No such file or directory
/opt/conda/lib/python3.8/site-packages/torch/distributed/launch.py:178: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use_env is set by default in torchrun.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
WARNING:torch.distributed.run:*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 313, in <module>
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 313, in <module>
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 313, in <module>
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 313, in <module>
                main()main()main()main()



  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 306, in main
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 306, in main
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 306, in main
  File "/home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py", line 306, in main
        t_start = time.time()t_start = time.time()

        t_start = time.time()t_start = time.time()

NameErrorNameErrorNameErrorNameError: : name 'time' is not definedname 'time' is not defined

: name 'time' is not defined
: name 'time' is not defined
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 125512) of binary: /opt/conda/bin/python3
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/conda/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/launch.py", line 193, in <module>
    main()
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/launch.py", line 189, in main
    launch(args)
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/launch.py", line 174, in launch
    run(args)
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/run.py", line 689, in run
    elastic_launch(
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 131, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/opt/conda/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
******************************************************************************************************************
  /home/hju/run_monaicore/multigpu/unetr_ddp/gputransform/unetr_btcv_ddp_gputransform_ThreadDataLoader.py FAILED  
==================================================================================================================
Root Cause:
[0]:
  time: 2022-05-03_16:02:44
  rank: 0 (local_rank: 0)
  exitcode: 1 (pid: 125512)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
==================================================================================================================
Other Failures:
[1]:
  time: 2022-05-03_16:02:45
  rank: 1 (local_rank: 1)
  exitcode: 1 (pid: 125546)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[2]:
  time: 2022-05-03_16:02:45
  rank: 2 (local_rank: 2)
  exitcode: 1 (pid: 126478)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[3]:
  time: 2022-05-03_16:02:45
  rank: 3 (local_rank: 3)
  exitcode: 1 (pid: 126497)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
******************************************************************************************************************

Run failed on c0909a-s17; retrying in 4 second(s)...
[33mWARNING:[0m underlay of /usr/bin/nvidia-smi required more than 50 (484) bind mounts
/.singularity.d/env/10-docker2singularity.sh: line 2: uname: No such file or directory
/opt/conda/lib/python3.8/site-packages/torch/distributed/launch.py:178: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use_env is set by default in torchrun.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
WARNING:torch.distributed.run:*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
slurmstepd: error: *** STEP 31647211.0 ON c0909a-s17 CANCELLED AT 2022-05-03T17:39:18 DUE TO TIME LIMIT ***
slurmstepd: error: *** JOB 31647211 ON c0909a-s17 CANCELLED AT 2022-05-03T17:39:18 DUE TO TIME LIMIT ***
